CHANGELOG.md

```diff
@@ -1,10 +1,18 @@
+## 2.14.0 (Feb, 3, 2022)
+FEATURES:
```

Comment:
We need an empty line between the version/date and the category, per TF guideline.

Reply:
done